### PR TITLE
Dapper base image

### PR DESCRIPTION
**Pull request by mkolesnik**
Copying logic to build and deploy the base dapper image to shipyard

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-TARGETS := $(shell ls scripts)
+TARGETS := $(shell ls scripts | grep -v dapper-image)
```
Comment (mangelajo) [inline Makefile]: In this case you can remove the grep -b dapper-image and L11:L13 right?
```suggestion
TARGETS := $(shell ls scripts)
```

File: Makefile [hunk 2]
```diff
@@ -8,6 +8,12 @@ TARGETS := $(shell ls scripts)
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
+dapper-image: .dapper
+	./.dapper -m bind dapper-image
+
```
Comment (mangelajo) [inline Makefile]:
```suggestion
```

File: .travis.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ git:
 
 jobs:
   include:
-  - env: CMD="make validate"
+  - env: CMD="make validate" DEPLOY=true
```
Comment (mangelajo) [inline .travis.yml]:
```suggestion
  - env: CMD="make validate dapper-image" DEPLOY=true
```

? or do we call the dapper-image part from validate? I can't remember, may be it was part of "ci"

Comment (mkolesnik) [reply .travis.yml]: you're right, I forgot to add it to the Makefile, thanks

File: Makefile (hunk 1 above)
Comment (mkolesnik) [reply Makefile]: I'd keep it for now for consistency with subm since we alrady have a validate target and will also have e2e and maybe some other ones

Comment (mkolesnik) [reply Makefile]: Actually makes sense.. I'll try it

Comment (skitt) [reply Makefile]: But Make knows how to deal with that on its own, without the `grep -v` and the extra rule — it breaks circular dependencies properly...

Comment (mkolesnik) [reply Makefile]: Yeah it seems that it doe, cool, it's gone then :)

**Pull request merged**